pkg/api/common/versions.go

```diff
@@ -357,8 +357,9 @@ var AllKubernetesSupportedVersionsAzureStack = map[string]bool{
 	"1.19.15": false,
 	"1.20.6":  false,
 	"1.20.11": false,
-	"1.21.10": true,
-	"1.22.7":  true,
+	"1.21.10": false,
+	"1.22.7":  false,
+	"1.22.8":  true,
```

Comment:
is this ok @haofan-ms?

Reply:
In AKSe v0.70.0, we support 1.22.7 aligning with upstream team. If the plan for new release is to support latest 1.22.9 (in L310), then we should follow that as well. But this is something we can control and update when we validate from Azure Stack side.